README.md

```diff
@@ -4,6 +4,7 @@
 
 > You will set up the environment and tools needed to develop a React application.
 > You will develop a React component that will hold the core functionality: a calculator.
+> Add the logic needed to make the Calculator component you developed in the previous project actually work
 
 
```

Comment:
### Your Getting Started Section
![Getting-Started2](https://user-images.githubusercontent.com/93039359/198348638-96c24ec7-2af1-4345-8efa-999f98be897c.PNG)

- [ ] As you can see the image above. That's how your Getting Started section looks like. Kindly remove all the commands in the Getting Started Section and add the following commands in it 👍 to make your readme descriptive 😊

```
## Getting Started

`git clone git@github.com:Iptysam/Math-magicians.git`
`cd Math-magicians`
`npm start`

```

Reply:
Thank you